### ISSUE TITLE
WiFi does not work correctly on Ubuntu

### ISSUE THREAD

**Issue by AkibAzmain** [Borwe/rt3290-linux-drivers#2]
When I start the computer, there is no active wifi interface and couldn't detect any wifi adapter, after activating the interface and restarting NetworkManager, the system recognises the adapter but couldn't find any wireless network. After hibernation and restarting at this state it works. But after I turn off and turn on WiFi, the system again can't find any wireless network. My kernel version is 5.0.0 and 5.3.0.

**Comment by Borwe** [comment]
what is the name of the new wifi interface after installing?
Did you try:
`sudo ifconfig eno1 up`
here eno1 being the interface name, might be different on yours.

**Comment by AkibAzmain** [comment]
Yes, I tried. My interface is rename3 and it changes if I unload and reload the module like this rename4, rename5...

**Comment by Borwe** [comment]
What kernel version exactly are you on? what does
`uname -r` 
show when run on terminal?

**Comment by AkibAzmain** [comment]
My kernel version is `5.3.0-51-generic` and `5.0.0-050000-generic` .

**Comment by AkibAzmain** [comment]
I have upgraded to Ubuntu 20.04 and now I don't have this problem. But I have some other problems. So I'm closing this issue.

**Comment by AkibAzmain** [comment]
When I was trying to find out the cause of new problem after upgrading, I found that this problem is the root of all problems. So, reopening.

**Comment by Borwe** [comment]
Did the problem have something to do with **/usr/src/linux-headers-5.4.0-42-generic/tools/build/Build.include** missing?
